### PR TITLE
Tutor command

### PR DESCRIPTION
**Pull request by Bishoy-at-pieces**
1. Tutor Pieces
2. PiecesFeedback
3. PiecesContribute
4. PiecesInstall

### CODE REVIEW

File: lua/pieces/feedbacks.lua [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    if choice == 1 then
+        vim.fn.PiecesOpenLink("https://github.com/pieces-app/plugin_neovim/discussions/66")
+    end
+end, {})
+
+vim.api.nvim_create_user_command('PiecesContribute', function()
+    local choice = vim.fn.confirm(
+[=[Contribute to the project
+https://github.com/pieces-app/plugin_neovim
+Would you like to open the feedback page in your browser?]=]
```
Comment (mack-at-pieces) [inline lua/pieces/feedbacks.lua:18]: Should this be a different string?

File: lua/pieces/tutor.lua [hunk 2]
```diff
@@ -15,11 +16,11 @@ Go to the Visual mode and select the following snippet:
 pip install pieces-cli
 ~~~
 
-Run `:PiecesCreateSnippet` to create a snippet
+Run `:PiecesCreateSnippet`{normal} to create a snippet
```
Comment (mack-at-pieces) [inline lua/pieces/tutor.lua:19]: @Bishoy-at-pieces do you mind sharing a screenshot of this whole output so I can see it from a users point of view?

File: plugin/pieces.lua [hunk 3]
```diff
@@ -1,3 +1,5 @@
 require("pieces.copilot")
 require("pieces.assets")
-require("pieces.copilot.slash_commands")
\ No newline at end of file
+require("pieces.copilot.slash_commands")
+require("pieces.feedbacks")
```
Comment (mack-at-pieces) [inline plugin/pieces.lua:4]: pieces.feedback

File: lua/pieces/tutor.lua (hunk 2 above)
Comment (Bishoy-at-pieces) [reply lua/pieces/tutor.lua:19]: Screenshot 2024-11-05 at 9 47 51 PM

File: lua/pieces/feedbacks.lua (hunk 1 above)
Comment (Bishoy-at-pieces) [reply lua/pieces/feedbacks.lua:18]: why? contributing should link to the repo right?

Comment (mack-at-pieces) [reply lua/pieces/feedbacks.lua]: I meant this part: "Would you like to open the feedback page in your browser?" It's not about feedback, it's about contributions. 

"Would you like to open GitHub?"

**Review by mack-at-pieces — APPROVED** [review]

**Pull request merged**